game/game.go

```diff
@@ -1142,6 +1143,27 @@ func asyncStartGame(ctx context.Context, gameID *datastore.Key, host string) err
 		}
 		g.ID = gameID
 
+		discordBotToken, err := auth.GetDiscordBotToken(ctx)
```

Comment:
The stuff happening here and in phase.go are _very_ similar. Perhaps extract them to some "call discord webhook with given data"-function?

Reply:
Updated. Let me know if you're happy with this approach and I'll merge. Have tested against local container and it works

Reply:
:+1: